site/docs/guidelines/tables-lists.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
+tags: ["Data Tables", "Tabular Data", "Cards", "Tiles", "Rows", "Grids"]
+inComparingSection: true
+---
+
+- **Tables also known as:** Data Tables, Tabular Data
+- **Lists also known as:** Cards, Tiles, Rows, Grids
+
+## Need to know
+### Tables
+- Tables are for presenting information which gives the user the flexibility to sort the data to find their own patterns
+- Tables must include column headers for sorting
+
+### Lists
+- Lists are for presenting grouped information into scannable patterns. The content within a list can be stacked to create hierarchy within the data.
+- Lists are sortable by a ‘Sorted by’ Select, and do not have header labels
+- A single row of a list could make sense
+
+### General
+- Tables and lists have distinct patterns which shouldn't be mixed
+- Lists and tables can be quite visually similar and can sometimes represent the same data. This guide will outline content and user considerations in determining which pattern is the most appropriate.  
+
+## Tables
+Tables display rows and columns of tabular data. 
+
+### For the user
+- They help make it easy to scan large amounts of data
+- Makes it efficient to find patterns in values
+- Data can be sorted by ascending or descending order using table column headers
+
+### For the designer
```

Comment:
I wonder if we can break up this list any further. There's a lot of bullet points here. Maybe subcategories could be: elements, structure, sorting... something like that.

Reply:
I've broken it up a bit, but we can iterate 👍